mrchecker-framework-modules/mrchecker-webapi-module/src/test/java/com/capgemini/mrchecker/webapi/httpbin/Base64Test.java

```diff
@@ -0,0 +1,41 @@
+package com.capgemini.mrchecker.webapi.httpbin;
+
+import com.capgemini.mrchecker.test.core.logger.BFLogger;
+import com.capgemini.mrchecker.webapi.BasePageWebApiTest;
+import com.capgemini.mrchecker.webapi.pages.httbin.Base64Page;
+import io.restassured.response.Response;
+import org.junit.Test;
+
+import java.text.MessageFormat;
+
+import static org.hamcrest.Matchers.is;
+import static org.junit.Assert.assertThat;
+
+public class Base64Test extends BasePageWebApiTest {
+	private Base64Page base64Page = new Base64Page();
+
+	@Test
+	public void validateCorrectnessOfBase64Decode_AwesomeText() {
+		String validBase64Data = "SFRUUEJJTiBpcyBhd2Vzb21l";
+		String validBase64Decode = "HTTPBIN is awesome";
+		base64DecodeDataCheck(validBase64Data, validBase64Decode);
+	}
+
+	@Test
+	public void validateCorrectnessOfBase64Decode_MrCheckerText() {
+		String validBase64Data = "TXIuQ2hlY2tlcg==";
+		String validBase64Decode = "Mr.Checker";
+		base64DecodeDataCheck(validBase64Data, validBase64Decode);
+	}
+
+	private void base64DecodeDataCheck(String base64Data, String expectedResult) {
```

Comment:
Please consider to use Parameters for this tests.
Example from https://www.testwithspring.com/lesson/writing-parameterized-tests-with-junit-4/:
```
@RunWith(JUnitParamsRunner.class)
@Category(UnitTest.class)
public class JUnitParamsCalculatorTest {
 
    @Test
    @Parameters({
            "0, 0, 0",
            "1, 1, 2"
    })
    public void shouldReturnCorrectSum(int first, 
                                       int second, 
                                       int expectedSum) {
        Calculator calculator = new Calculator();
        int actualSum = calculator.sum(first, second);
        assertThat(actualSum).isEqualByComparingTo(expectedSum);
    }
}
```